Sources/DeepLearning/Optimizer.swift

```diff
@@ -129,8 +129,7 @@ public class SGD<Model: Layer, Scalar: TensorFlowFloatingPoint>: Optimizer
         momentum: Scalar = 0,
         decay: Scalar = 0,
         nesterov: Bool = false,
-        modelType: Model.Type = Model.self,
-        scalarType: Scalar.Type = Scalar.self
+        for model: Model
```

Comment:
* Replace unused argument names with an underscore.
* Use `__shared` to avoid a semantic copy.
```suggestion
        for _: __shared Model
```

Reply:
Fixed.